### PR TITLE
:sparkles: Use Node.js binary

### PR DESCRIPTION
**Pull request by blakek**
This transitions to a self-contained Node.js binary that installs everything.  The reasoning being:

- more people on our team know JavaScript than Bash
- we can do things without relying on the environment as much
- adding new features (e.g. optional installs) is easier now

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  const response = await axios.get(
+    'https://raw.githubusercontent.com/Homebrew/install/master/install'
+  );
+
+  const installScript = response.data;
+
+  await execute('ruby', ['-e', installScript]);
+
+  return executeShell(`
+    # Ensure folder for n exists
```
Comment (blakek) [inline src/index.js:76]: I admit I cheat a bit and use `executeShell()`.  While doing this in Node is possible, I think it's just too much for what it does.

File: .babelrc.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+		[
+			'@babel/preset-env',
+			{
+				targets: {
+					node: 'current'
+				}
+			}
+		]
+	]
+};
```
Comment (zstickles-gsandf) [inline .babelrc.js:12]: I know we're using tabs for the bash, but it looks like we're using spaces for JS. Maybe we should use spaces here.

File: package.json [hunk 3]
```diff
@@ -1,14 +1,41 @@
...
+  "author": "GS&F Devs <mindreactor@gsandf.com> (https://gsandf.com/)",
+  "license": "MIT",
+  "files": [
+    "src/**",
+    "dist/**"
+  ],
+  "eslintConfig": {
+    "extends": "gsandf",
+    "parser": "babel-eslint"
+  },
```
Comment (zstickles-gsandf) [inline package.json:16]: Out of curiosity, why did you opt for setting this in the `package.json` instead of it's own file?

**Review by zstickles-gsandf — APPROVED** [review]

File: .babelrc.js (hunk 2 above)
Comment (blakek) [reply .babelrc.js:12]: 🛠 fixed it

File: package.json (hunk 3 above)
Comment (blakek) [reply package.json:16]: No reason

**Review by SethGunnells — APPROVED** [review]

**Pull request merged**